doc/api/errors.md

```diff
@@ -255,6 +255,13 @@ will affect any stack trace captured *after* the value has been changed.
 If set to a non-number value, or set to a negative number, stack traces will
 not capture any frames.
 
+#### error.code
```

Comment:
I am not sure whether this section belongs here...at least we need to mention that this property doesn't appears in all `Error`s, and it is set by Node.js?